### ISSUE TITLE
Grid lines - add option to set thickness

### ISSUE THREAD

**Issue by havatv** [OpenOrienteering/mapper#733]
When printing using 0.6.3 (on Ubuntu and Windows), the width of grid lines seems to scale according to map scale.  On screen, the width of the grid lines do not scale when zooming.
When printing base maps, grid lines are useful for referencing, but they should be thin in order not to interfere with the base map.
It would be useful to be able to specify the printed width of the grid lines.  Preferably a scale-independent width.
This seems to be related to https://github.com/OpenOrienteering/mapper/issues/227, but the fix for that issue does not seen to solve this problem.

**Comment by dg0yt** [comment]
Grid lines are printed with 0.1 mm at map scale which is thin. But they scale with printing scale. Right?

**Comment by havatv** [comment]
Yes.  When mapping according to ISOM with larger scales than normal (e.g. 1:5000 and larger) and base maps with a lot of detail, the the grid lines get too thick and tend to hide things.  One option is to keep the 0.1 mm width but not scale the width of the grid lines when changing map scale for printing, another option is to make the thickness configurable.

**Comment by dg0yt** [comment]
A major sub-issue is fixed, but I'm not planning to work on custom grid line width soon.

**Comment by havatv** [comment]
Avoiding scaling is a real improvement - thanks! 
Being able to specify the width of the grid lines is not that important, so it is OK for me to close this issue now.

**Comment by dg0yt** [comment]
I don't want to close it. It is a reasonable feature request. (It is even not very difficult to implement, but of course it touches GUI as well as the file format.) I just had to clarify that this feature won't be part of v0.8.4.

**Comment by havatv** [comment]
@dg0yt, was it closed accidentally (closing is not consistent with https://github.com/OpenOrienteering/mapper/issues/733#issuecomment-446075374 that you wrote two days ago)?

**Comment by dg0yt** [comment]
Thanks for the hint. Commit 81f6da0 states that it fixes this issue and triggered an automatism. However,  this commit statement was somewhat inaccurate.

**Comment by dg0yt** [comment]
The master and unstable builds now contain the line scaling fix.
